### ISSUE TITLE
Mention License of font files

### ISSUE THREAD

**Issue by balasankarc** [miketierney/artii#9]
Hi,
I am trying to package artii to Debian. Can you please specify the license of font files included in `lib/figlet/fonts` somewhere?

**Comment by miketierney** [comment]
Absolutely; sorry for the delay. This message apparently was lost in my inbox; I hope it's not caused you too much trouble. I'll try to get it updated this week.

**Comment by olleolleolle** [comment]
@miketierney Could [this text document from the Figlet codebase](https://github.com/cmatsuoka/figlet/blob/master/figfont.txt#L147) contain the information you need? Hm, maybe maybe not.

[the emboss font in 2006 seems to have had another license](https://github.com/cmatsuoka/figlet/blob/a565ae1e8f8254044219260dda2a6b51984930dc/tests/emboss.tlf#L8), but the above link is about later developments (2011 and later). Did the Figlet license collation miss a spot?

**Comment by miketierney** [comment]
I apologize for the incredibly slow turn-around on this; this project hasn't had my attention like it's deserved, but it's not forgotten. I won't promise a specific timeline for addressing this (so as to not disappoint), I will make every effort to address this issue as soon as I'm able.

**Comment by olleolleolle** [comment]
@miketierney _Thank you_ for the project. As long as you enjoy doing it, you should do it.

**Comment by balasankarc** [comment]
@miketierney No worries. Thanks a lot for the project. :)
